system/Test/Fabricator.php

```diff
@@ -545,6 +548,13 @@ public function create(int $count = null, bool $mock = false)
 				$ids[] = $id;
 				self::upCount($this->model->table);
 			}
```

Comment:
probably bettter to use `continue` statement in previous `if`, and remove `else`

Reply:
Yes, the `continue` would be better here.

Reply:
Will do!